contracts/cw721-balance-voting/schema/count_response.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Looks like we need to regenerate these schema files. :) Should be able to run `cargo schema` in the root for that.